react/tic-tac-toe/src/app/screens/Login/Validations/index.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line no-useless-escape
+const VALID_EMAIL = /^\w+([\.-]?\w+)*@\wolox.com/;
+
+exports.isRequired = value => value ? value : 'Required field';
+
+exports.hasLenght = value => value.lenght > 8 ? value : 'Value lenght has to be greater than 8 characters';
```

Comment:
Same. 

Also, we set text as constants so you can re use them through out all the app. It's not necessary for you to change it here, but keep that in mind. We also use i18next  dependency for internationalization in our projects

Reply:
gotcha